cmd/eth1indexer/main.go

```diff
@@ -716,7 +716,25 @@ func IndexFromNode(bt *db.Bigtable, client *rpc.ErigonClient, start, end, concur
 
 	}
 
-	return g.Wait()
+	err := g.Wait()
+
+	if err != nil {
+		return err
+	}
+
+	lastBlockInCache, err := bt.GetLastBlockInBlocksTable()
+	if err != nil {
+		return err
+	}
+
+	if end > int64(lastBlockInCache) {
```

Comment:
This will not work if redis has not yet been set.
As GetLastBlockInBlocksTable will return the same block that has just been added to to bt by the importer so end will be the same as lastBlockInCache